runtime/src/bank.rs

```diff
@@ -284,8 +279,8 @@ pub struct Bank {
     /// The number of slots per year, used for inflation
     slots_per_year: f64,
 
-    /// The number of slots per Storage segment
-    slots_per_segment: u64,
+    /// Unused
+    unused: u64,
```

Comment:
@sakridge Can we remove these without affecting reply/bank hash?

Reply:
I don't think bank hash would be affected, but the bank is serialized into the snapshot, so that wouldn't be ABI compatible anymore if this was removed.